compiler/luci/service/src/CircleTypeInferenceRule.cpp

```diff
@@ -589,6 +594,13 @@ struct TypeInferenceAlgorithm final : public luci::CircleNodeVisitor<loco::DataT
     return loco::DataType::S32;
   }
 
+  loco::DataType visit(const luci::CircleNonMaxSuppressionV5Out *node) final
+  {
+    (void)node;
+    assert(node->index() >= 0 || node->index() <= 2);
+    return loco::DataType::S32;
```

Comment:
TensorFlow op
```
REGISTER_OP("NonMaxSuppressionV5")
    .Input("boxes: T")
    .Input("scores: T")
    .Input("max_output_size: int32")
    .Input("iou_threshold: T")
    .Input("score_threshold: T")
    .Input("soft_nms_sigma: T")
    .Output("selected_indices: int32")
    .Output("selected_scores: T")
    .Output("valid_outputs: int32")
    .Attr("T: {half, float} = DT_FLOAT")
    .Attr("pad_to_max_output_size: bool = false")
```
Second op ` .Output("selected_scores: T")` is not `int32`  . Can you provide where I could reference that all output types are `S32`?

Reply:
Actually, it's float32 in tflite kernel, I've used this file for reference: https://github.com/tensorflow/tensorflow/blob/2a96849f478627ad03b8f20f8c71e85d6d7480e3/tensorflow/lite/kernels/non_max_suppression.cc#L58

I've updated this section to use float32, thank you for this note :)